backend/experiment/views.py

```diff
@@ -175,20 +180,42 @@ def serialize_experiment_series(
     }
 
 
-def serialize_experiment(experiment_object: Experiment, finished=0):
+def serialize_experiment(experiment_object: Experiment, started=0, finished=0):
```

Comment:
See #933: need to pass in the `participant` here, and call the `get_started_session_count` function to retrieve the number. Otherwise, the dashboard will always display `0`.

Reply:
Oh, I liked this approach of keeping the database out of the serializer, but sure 😄 Maybe it's a good idea to look at that in the future and try to create some separation between aggregating the data needed for serialization and the actual serialization part.